@navikt/core/react/src/tooltip/Tooltip.tsx

```diff
@@ -184,18 +192,22 @@ export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
         );
     }
 
+    const labelProps = describeChild
+      ? _open
+        ? { "aria-describedby": ariaId }
+        : { title: content }
```

Comment:
Edge case: If the delay is too long, the native title tooltip might show up before the `title` prop is removed. Also, we never know if any of the browsers changes the delay of the native tooltip.

I think ideally, the `title` should be removed immediately on hover, but I bet that would be cumbersome. We could also warn about it in the JSDOC or with console.warning().

On the other hand, I think it's unlikely that devs will increase the delay significantly. And even if the edge case happens, it's not really a big problem.

Reply:
I think this is one of those edgecases where its not optimal as is, but as you said only an issue if they manually override it to 500ms++. So i think this is a wontfix.